ports/esp32/mpconfigport.h

```diff
@@ -249,6 +249,7 @@ struct mp_bluetooth_nimble_root_pointers_t;
     const char *readline_hist[8]; \
     mp_obj_t machine_pin_irq_handler[40]; \
     struct _machine_timer_obj_t *machine_timer_obj_head; \
+    struct _machine_i2s_obj_t *machine_i2s_obj[2]; \
```

Comment:
Would be good to use `I2S_NUM_MAX` here instead of 2 (is that possible?.

Reply:
Yes!.  I submitted a commit that uses the ESP-IDF MACRO rather than the magic number.  I'm really glad you mentioned this.  A couple of months ago I attempted to add the I2S root pointer and saw many compile errors when I included the supporting ESP-IDF driver header file.  Today I see no issues.